### ISSUE TITLE
NEP141 ft_metadata not working

### ISSUE THREAD

**Issue by mfornet** [aurora-is-near/aurora-engine#278]
When calling ft_metadata response contains invalid json:

```
❯ near view aurora ft_metadata '{}'
View call: aurora.ft_metadata({})
An error occured
SyntaxError: Unexpected token } in JSON at position 4149
    at JSON.parse (<anonymous>)
    at parseJsonFromRawResponse (/usr/local/lib/node_modules/near-cli/node_modules/near-api-js/lib/account.js:24:17)
    at Account.viewFunction (/usr/local/lib/node_modules/near-cli/node_modules/near-api-js/lib/account.js:361:61)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async exports.callViewFunction (/usr/local/lib/node_modules/near-cli/index.js:75:48)
    at async Object.handler (/usr/local/lib/node_modules/near-cli/utils/exit-on-error.js:52:9)
SyntaxError: Unexpected token } in JSON at position 4149
    at JSON.parse (<anonymous>)
    at parseJsonFromRawResponse (/usr/local/lib/node_modules/near-cli/node_modules/near-api-js/lib/account.js:24:17)
    at Account.viewFunction (/usr/local/lib/node_modules/near-cli/node_modules/near-api-js/lib/account.js:361:61)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async exports.callViewFunction (/usr/local/lib/node_modules/near-cli/index.js:75:48)
    at async Object.handler (/usr/local/lib/node_modules/near-cli/utils/exit-on-error.js:52:9)
```

This is blocking having nETH in the wallet.

**Comment by joshuajbouw** [comment]
I've assigned this to @mrLSD to get in.

**Comment by birchmd** [comment]
The issue is that the [`Display` implementation for our `JsonValue` type](https://github.com/aurora-is-near/aurora-engine/blob/develop/src/json.rs#L253-L257) does not produce valid json. It contains a trailing comma on the last field, which is invalid in the json spec.

Unfortunately this issue was not caught at the time we were adding `ft_metadata`, [even though we have a test for it](https://github.com/aurora-is-near/aurora-engine/blob/develop/src/tests/sanity.rs#L432-L446), because the `rjson` parser apparently allows trailing commas.

I think the best solution here is to change `JsonValue` `Display` implementation to produce valid json.

**Comment by birchmd** [comment]
Fixed in #279
